.gitignore

```diff
@@ -128,3 +128,5 @@ keys/
 .vscode/
 
 ci/terraform/.terraform
+/credentials.json
```

Comment:
What credentials...? I renamed the previously used `credentials.json` to `googleCredentials.json`

If this is not needed, remove it. Otherwise name it more specifically.